src/test/java/uk/gov/digital/ho/hocs/audit/aws/listeners/AuditListenerTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.digital.ho.hocs.audit.aws.listeners;
+
+import com.google.gson.Gson;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.digital.ho.hocs.audit.AuditDataService;
+
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class AuditListenerTest {
+
+    @Autowired
+    private Gson gson;
+
+    @Mock
+    private AuditDataService auditDataService;
+
+    @Test
+    public void callsAuditServiceWithValidCreateCaseMessage() {
```

Comment:
can we write a similar test where we put in an invalid format message onto the queue.

Reply:
This has been added 👍🏽